applications/asset_tracker_v2/src/cloud/cloud_codec/Kconfig

```diff
@@ -4,6 +4,34 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
+config GPS_BUFFER_MAX
+	int "GPS data ringbuffer entries"
+	default 15
```

Comment:
Buffer access is master if the item count is a power of 2

Reply:
Im not sure I quite understood the sentence here.

Reply:
Oops, typo, corrected now..

Reply:
Ah, ok. I see. I can add a comment and correct the entries to 16.